scripts/build.sh

```diff
@@ -92,7 +94,7 @@ cmake $ADDITIONAL_CMAKE_OPTIONS \
   ${COREMLTOOLS_HOME}
 
 # Make the python wheel
-make -j${NUM_PROCS}
+make -j${NUM_PROCS} VERBOSE=TRUE
```

Comment:
Probably should remove VERBOSE=TRUE (it's only needed while debugging the CI configuration and it bloats the log output).